lib/std/rand.zig

```diff
@@ -27,6 +27,7 @@ pub const Pcg = @import("rand/Pcg.zig");
 pub const Xoroshiro128 = @import("rand/Xoroshiro128.zig");
 pub const Xoshiro256 = @import("rand/Xoshiro256.zig");
 pub const Sfc64 = @import("rand/Sfc64.zig");
+pub const Romu192 = @import("rand/Romu192.zig");
```

Comment:
Perhaps rename this to `RomuTrio`, as that is the commonly-recognised and used name for the algorithm.

Reply:
@francescoalemanno Just making sure you didn't miss these suggestions

Reply:
done